homeassistant/components/advantage_air/climate.py

```diff
@@ -84,34 +103,56 @@ class AdvantageAirAC(AdvantageAirAcEntity, ClimateEntity):
     _attr_min_temp = 16
     _attr_name = None
     _enable_turn_on_off_backwards_compatibility = False
+    _support_preset = ClimateEntityFeature(0)
 
     def __init__(self, instance: AdvantageAirData, ac_key: str) -> None:
         """Initialize an AdvantageAir AC unit."""
         super().__init__(instance, ac_key)
 
-        self._attr_supported_features = (
-            ClimateEntityFeature.FAN_MODE
-            | ClimateEntityFeature.TURN_OFF
-            | ClimateEntityFeature.TURN_ON
-        )
-        self._attr_hvac_modes = [
-            HVACMode.OFF,
-            HVACMode.COOL,
-            HVACMode.HEAT,
-            HVACMode.FAN_ONLY,
-            HVACMode.DRY,
-        ]
-        # Set supported features and HVAC modes based on current operating mode
+        self._attr_preset_modes = [ADVANTAGE_AIR_MYZONE]
```

Comment:
Can this change at runtime? Should it be in the new functions?

Reply:
No, the preset modes avaliable should never change.